spotlight-client/src/PageNarrative/PageNarrative.tsx

```diff
@@ -0,0 +1,50 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+import { RouteComponentProps } from "@reach/router";
+import { observer } from "mobx-react-lite";
+import React from "react";
+import { SystemNarrativeTypeId } from "../contentApi/types";
+import { useDataStore } from "../StoreProvider";
+import withRouteSync from "../withRouteSync";
+
+type PageNarrativeProps = RouteComponentProps & {
+  narrativeTypeId?: SystemNarrativeTypeId;
+};
+
+const PageNarrative: React.FC<PageNarrativeProps> = ({ narrativeTypeId }) => {
+  const tenant = useDataStore().tenantStore.currentTenant;
```

Comment:
meaning, like, it would be `useDataStore().currentTenant` instead of this? yeah maybe that's a good idea ... I keep writing this line over and over again

Reply:
Or `useTenantStore().currentTenant`

Reply:
oh i see what you meant .... I _think_ it is simpler for there to just be one context that contains the entire data store? that is sort of what they recommend but maybe it is not strictly necessary ... i haven't tried that